scss/navbar.scss

```diff
@@ -0,0 +1,24 @@
+.navbar {
+    height: 90px;
+    background: $white;
+    position: fixed;
+    z-index: 10;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    -webkit-box-shadow: 0px 5px 5px 0px rgba(0,0,0,0.75);
+    -moz-box-shadow: 0px 5px 5px 0px rgba(0,0,0,0.75);
+    box-shadow: 0px 5px 5px 0px rgba(0,0,0,0.75);
+  
+.nav-image1 {
```

Comment:
add a '-' between image and 1. likewise, you should be more descriptive naming classes